src/SixLabors.Fonts/Unicode/BidiData.cs

```diff
@@ -172,4 +174,15 @@ public ArraySlice<sbyte> GetTempLevelBuffer(int length)
         this.tempLevelBuffer.Clear();
         return this.tempLevelBuffer.Add(length, false);
     }
+
+    public void Free()
```

Comment:
From C# 8 onwards ref structs can add Dispose methods which mean the compiler will allow `using` statements.

Reply:
But ref struct can't import interface. Is that means I can announce a method called Dispose, so I can use using statements?

Reply:
Yes, they don't implement the interface but the compiler spots the pattern and allows you to use it.